### PR TITLE
feat: Front Lead Two with ITE

### PR DESCRIPTION
**Pull request by Geoff-Ford**
https://nidigitalsolutions.jira.com/browse/TNLT-5064

* Created a new Front Lead Two Slice and associated Tiles
* Fixed Slice Layout widths
* Applied above fix to Front Lead One

### iPad Mini 4 - Portrait
iPad_mini_4 _Portrait_(768)

### iPad Mini 4 - Landscape
iPad_mini_4 _Landscape_(1024)

### iPad Air - Portrait
iPad_Air _Portrait_(834)

### iPad Air - Landscape
iPad_Air _Landscape_(1112)

### iPad - Portrait
iPad _Portrait_(810)

### iPad - Landscape
iPad_Landscape_(1080)

### iPad Pro (11) - Portrait
iPad_Pro_(11)_Portrait_(834)

### iPad Pro (11) - Landscape
iPad_Pro_(11)_Landscape_(1194)

### iPad Pro (12.9) - Portrait
iPad_Pro_(12 9)_Portrait_(1024)

### iPad Pro (12.9) - Landscape
iPad_Pro_(12 9)_Landscape_(1366)

### CODE REVIEW

File: packages/slice-layout/src/templates/frontleadtwo/styles.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+          ...inTodaysEditionContainerPortrait,
+          height: 148,
+        },
+      },
+      "1024": {
+        ...sharedPortraitStyles,
+        container: {
+          ...container,
+          paddingTop: spacing(6),
+          paddingBottom: spacing(6),
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/frontleadtwo/styles.js:163]: `verticalPadding`

File: packages/styleguide/src/grid/columnToPercentage.ts [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
   totalColumns = 12,
 }) => {
-  const colWidth = 100;
-  const gapWidth = 50;
+  const windowWidth = Dimensions.get("window").width;
+  const contentWidth = calculateContentWidth(windowWidth, orientation);
 
-  const totalWidthOfAllColumns =
-    totalColumns * colWidth + (totalColumns - 1) * gapWidth;
+  const gapWidth = 21;
```
Comment (aronshamash) [inline packages/styleguide/src/grid/columnToPercentage.ts:22]: `spacing(4) + 1`

File: packages/styleguide/src/grid/columnToPercentage.ts [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
+  const marginWidth = 10;
```
Comment (aronshamash) [inline packages/styleguide/src/grid/columnToPercentage.ts:23]: `spacing(2)`

File: packages/styleguide/src/grid/columnToPercentage.ts [hunk 4]
```diff
@@ -1,30 +1,38 @@
...
+  const allGapsWidth = (totalColumns - 1) * gapWidth;
```
Comment (aronshamash) [inline packages/styleguide/src/grid/columnToPercentage.ts:24]: `totalGapWidth` ?

File: packages/styleguide/src/grid/columnToPercentage.ts [hunk 5]
```diff
@@ -1,30 +1,38 @@
...
+  const allColumnsWidth = contentWidth - allGapsWidth;
```
Comment (aronshamash) [inline packages/styleguide/src/grid/columnToPercentage.ts:25]: `totalColumnWidth`

**Review by aronshamash — APPROVED** [review]
few minors looks good!

File: packages/edition-slices/__tests__/android/__snapshots__/tablet-slices.test.js.snap [hunk 6]
```diff
@@ -762,97 +762,60 @@ exports[`22. puzzle - medium 1`] = `
...
-          tileName="support2"
-        />
-      </View>
-    </View>
   </View>
 </View>
 `;
 
-exports[`24. front lead two no pic and two - portrait - medium 1`] = `
+exports[`24. front lead two - landscape - medium 1`] = `
```
[the remark this answers is not in the document]
Comment (Geoff-Ford) [reply packages/edition-slices/__tests__/android/__snapshots__/tablet-slices.test.js.snap:794]: Well these tests are ran agains all the included slices in `packages/edition-slices/__tests__/shared-tablet-slices.base.js`. I see the `mockLeadOneFullWidthFrontSlice` is also still in there and @c-j-j added the `sliceProps` to it. Are you suggesting we don't include the fronts in these tests?

File: packages/styleguide/src/grid/columnToPercentage.ts (hunk 2 above)
Comment (Geoff-Ford) [reply packages/styleguide/src/grid/columnToPercentage.ts:22]: Nice

File: packages/styleguide/src/grid/columnToPercentage.ts (hunk 4 above)
Comment (Geoff-Ford) [reply packages/styleguide/src/grid/columnToPercentage.ts:24]: I went with the same format as `allColumnsWidth` which was to distinguish it more clearly from `totalColumns`

File: packages/styleguide/src/grid/columnToPercentage.ts (hunk 5 above)
Comment (Geoff-Ford) [reply packages/styleguide/src/grid/columnToPercentage.ts:25]: see above

**Pull request merged**